ui/src/app/tasks/task-definition-create/task-definition-create.component.ts

```diff
@@ -107,34 +107,36 @@ export class TaskDefinitionCreateComponent implements OnInit, OnDestroy {
           severity: Flo.Severity[m.severity].toLowerCase()
         }))
       );
-      const dslText = this.dsl;
-      this.toolsService.parseTaskTextToGraph(dslText).toPromise().then(taskConversion => {
-        if (taskConversion.errors) {
-          taskConversion.errors.forEach(e => annotations.push({
-            from: this.getPosition(dslText, e['position']),
-            to: e['length'] ? this.getPosition(dslText, e['position'] + e['length'])
-              : this.getPosition(dslText, e['position'] + 1),
-            message: e['message'],
-            severity: 'error'
-          }));
-        }
+      const doc = editor.getDoc();
+      if (doc.lineCount() > 1) {
+        // Multiline definitions are not supported. Parser throws errors
+        annotations.push({
+          from: doc.posFromIndex(0),
+          to: doc.posFromIndex(doc.getValue().length - 1),
+          message: 'Multi-line composed task definitions are not supported',
```

Comment:
Change to `Multi-line task definitions are not supported`. Since the former `Composed Task` editor is now **THE** editor for ALL tasks, we probably should not use the term `composed` in too many contexts.